modules/builds-identifying-image-change-triggers.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * builds/triggering-builds-build-hooks.adoc
+
+[id="builds-image-change-trigger-identification_{context}"]
+= Indentifying the image change trigger of a build
+
+As a developer, if you have image change triggers, you can identify which image change initiated the last build. This can be useful for debugging or troubleshooting builds.
+
+.Example `BuildConfig`
+[source,yaml]
+----
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+  metadata:
+    name: bc-ict-example
+    namespace: bc-ict-example-namespace
+  spec:
+
+    ...
+
+    triggers:
+    - imageChange:
+        from:
+          kind: ImageStreamTag
+          name: input:latest
+          namespace: bc-ict-example-namespace
```

Comment:
Well, now that you've jumped into explictly citing multiple triggers being present, you example here should have multiple triggers :-)

Add after this line : 
```
- imageChange:
    from:
      kind: ImageStreamTag
      name: input2:latest
      namespace: bc-ict-example-namespace
   type:  ImageChange
```